components/DataPage/MechCategorizedRequests.jsx

```diff
@@ -0,0 +1,72 @@
+import { SUB_HEADER_LG_CLASS, TEXT_MEDIUM_CLASS } from 'common-util/classes';
+import { MECH_AGENT_CLASSIFICATION } from 'common-util/constants';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { ExternalLink } from 'components/ui/typography';
+import { CodeSnippet } from './CodeSnippet';
+
+const allMechIds = [
+  ...MECH_AGENT_CLASSIFICATION.predict,
+  ...MECH_AGENT_CLASSIFICATION.contribute,
+  ...MECH_AGENT_CLASSIFICATION.governatooor,
+];
+const IDS_LIST = JSON.stringify(allMechIds.map(String));
+
+const ONCHAIN_QUERY = `query {
+  requestsPerAgentOnchains(where: { id_in: ${IDS_LIST} }) {
+    id
+    requestsCount
+  }
+}`;
+
+const MARKETPLACE_QUERY = `query {
+  requestsPerAgents(where: { id_in: ${IDS_LIST} }) {
+    id
+    requestsCount
+  }
+}`;
```

Comment:
we can simply import from graphql/queries, no? also would be no need to map and stringify